protocol/testing/genesis.sh

```diff
@@ -26,8 +26,8 @@ ETH_CHAIN_ID=11155111 # sepolia
 # https://sepolia.etherscan.io/address/0xcca9D5f0a3c58b6f02BD0985fC7F9420EA24C1f0
 ETH_BRIDGE_ADDRESS="0xcca9D5f0a3c58b6f02BD0985fC7F9420EA24C1f0"
 BRIDGE_MODACC_BALANCE=1000000000000000000000000000 # 1e27
-BRIDGE_GENESIS_ACKNOWLEDGED_NEXT_ID=0 # TODO(CORE-329)
-BRIDGE_GENESIS_ACKNOWLEDGED_ETH_BLOCK_HEIGHT=0 # TODO(CORE-329)
+BRIDGE_GENESIS_ACKNOWLEDGED_NEXT_ID=0
```

Comment:
To confirm, what was the original TODO and why is it okay to remove the comments?

Reply:
original TODO is for us to determine these values before software release. however, by default these values can be just 0 and it's up to operators and maintainers of the chain to use the script [here](https://github.com/dydxprotocol/v4-chain/blob/main/protocol/scripts/bridge_events/bridge_events.go) to determine these values